### PR TITLE
Fix #1513: incorrect plural form for last inspection time

### LINKED ISSUE #1513: Incorrect plural form for last inspection time
2015-06-28 21_13_37-openrct2
Should be "1 minute"

### PR DESCRIPTION
**Pull request by mdlayher**
This seemed like an easy enough bug to solve, but there may also be an easier way instead of adding a whole new string for each language.  I am a little fuzzy on some of the translations as well.

My editor automatically trims trailing whitespace; please pardon the diff. I recommend using this link: https://github.com/OpenRCT2/OpenRCT2/pull/1672/files?w=1

Fixes #1513.

### CODE REVIEW

**Comment by IntelOrca** [comment]
You should only update english_uk.txt, the other languages are updated in the localisation repo and only when they have translations as strings now fallback to English UK anyway.

Can you also fix the commit so that you don't remove all these trailing whitespaces?

**Comment by mdlayher** [comment]
@IntelOrca Updated.

File: src/windows/ride.c [hunk 1]
```diff
@@ -3584,9 +3584,16 @@ static void window_ride_maintenance_paint(rct_window *w, rct_drawpixelinfo *dpi)
 
 	// Last inspection
 	lastInspection = ride->last_inspection;
-	stringId = lastInspection <= 240 ?
-		STR_TIME_SINCE_LAST_INSPECTION_MINUTES :
-		STR_TIME_SINCE_LAST_INSPECTION_MORE_THAN_4_HOURS;
+
+	// Use correct, singular form for 1 minute of time or less
+	// https://github.com/OpenRCT2/OpenRCT2/issues/1513
+	if (lastInspection <= 60)
```
Comment (IntelOrca) [inline src/windows/ride.c]: I think lastInspection is in minutes, so this should be ```<= 1```

Comment (mdlayher) [inline src/windows/ride.c]: Wow, can't believe I somehow translated "240 seconds" into "4 hours" a couple lines below.  My apologies.  I'll update this.

**Comment by Gymnasiast** [comment]
There are merge conflicts, you'll need to rebase it.

**Comment by Gymnasiast** [comment]
You merged it in quite an ugly way. I have put up a PR that has your edits, but merged in a much cleaner fashion.

**Pull request closed without merging**